src/services/completions.ts

```diff
@@ -1731,7 +1731,14 @@ function createCompletionEntry(
     // We should only have needsConvertPropertyAccess if there's a property access to convert. But see #21790.
     // Somehow there was a global with a non-identifier name. Hopefully someone will complain about getting a "foo bar" global completion and provide a repro.
     else if ((useBraces || insertQuestionDot) && propertyAccessToConvert) {
-        insertText = useBraces ? needsConvertPropertyAccess ? `[${quotePropertyName(sourceFile, preferences, name)}]` : `[${name}]` : name;
+        if (useBraces && preferences.includeCompletionsWithSnippetText) {

```

Comment:
Noble try but this diff doesn't work in VS Code. Revert this section of diff but leave the part that sets the sort order